kpis/kpi_calculator.py

```diff
@@ -532,6 +532,44 @@ def get_computational_time_ratio(self, plot=False):
 
         return time_rat
 
+    def _set_last_index(self,label):
```

Comment:
Actually, I think label is ok here because it actually does not reference a test case point name, rather the suffix used for the variable that keeps track of the last index in the kpi calculator.  The description was incorrect (not updated from an earlier version of the function I had tried out).  I've updated the description in this commit: https://github.com/ibpsa/project1-boptest/pull/290/commits/e38a5e62353d82e30f2b27b3b08bd1a601a715fc.

Reply:
To your general point though, I agree.  I've changed the name of the variable in ``_get_data_from_last_index`` accordingly in https://github.com/ibpsa/project1-boptest/pull/290/commits/7d63d36db8b5fd8ba53d7345f426827197638b96.